test-requirements.txt

```diff
@@ -107,29 +89,25 @@ pycparser==2.21
 pygments==2.17.2
     # via sphinx
 pylint==3.0.3
-    # via -r test-requirements.in
 pyopenssl==24.0.0
-    # via -r test-requirements.in
 pyproject-hooks==1.0.0
     # via build
 pyright==1.1.349
-    # via -r test-requirements.in
 pytest==8.0.0
-    # via -r test-requirements.in
 pytz==2024.1
     # via babel
 requests==2.31.0
     # via sphinx
 ruff==0.2.1
-    # via -r test-requirements.in
+setuptools==69.1.0
```

Comment:
Idk why pip-tools doesn't allow setup tools either, if it's a good reason then uv also shouldn't

Reply:
`pip-tools` is going to change their default soon to include it 🤷‍♀️ 

https://github.com/astral-sh/uv/issues/1353

Reply:
Alright then this change is fine